src/AllFilters.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Set;
+
+public class AllFilters {
+    private final PlusFilter plusFilter;
```

Comment:
Private fields only used in one method should be in method parameters list.

Reply:
sorry I didn't get your point